discord/ext/commands/bot.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, command_prefix, help_command=_default, description=None, **op
         self._help_command = None
         self.description = inspect.cleandoc(description) if description else ''
         self.owner_id = options.get('owner_id')
+        self.team_member_ids = {}
```

Comment:
Personally I feel as if using `owner_ids` as a key would have been better, along with making it possible to pass it at `__init__`. That way those who are not using teams for whatever reason can also just pass in some iterable or list and the library can cast it to a `set` without hitting any endpoint much like the current `owner_id` today.

This will probably require a check to make sure the following post-conditions are true:

- `owner_id` and `owner_ids` are not both set
- `owner_ids` can successfully be cast to an `set` or `int`.

Also requires changes to the stuff below.

Reply:
Why would `owner_ids` be able to be casted to an `int`?

Reply:
It's a typo -- I meant `set` of `int` not "or".